meshnet/server_test.go

```diff
@@ -58,8 +59,13 @@ func (n *workingNetworker) AllowFileshare(address UniqueAddress) error {
 	return nil
 }
 
-func (n *workingNetworker) AllowIncoming(address UniqueAddress) error {
+func (n *workingNetworker) AllowIncoming(address UniqueAddress, lanAllowed bool) error {
 	n.allowedIncoming = append(n.allowedIncoming, address)
+
+	if !lanAllowed {
```

Comment:
This mock implementation is becoming too big. Generally, you shouldn't care about the internal state of mocked object in unit tests. Actual internal state of actual networker can be tested in networker's unit tests. But for using it in e. g. meshnet server you only care that this method was called correctly.